tests/torch/test_knowledge_distillation.py

```diff
@@ -269,3 +278,71 @@ def test_kd_sparsity_statistics(algo: str):
            getattr(statistics_with_kd, algo).model_statistics.sparsity_level
     assert getattr(statistics, algo).model_statistics.sparsity_level_for_layers ==\
            getattr(statistics_with_kd, algo).model_statistics.sparsity_level_for_layers
+
+
+@pytest.mark.parametrize("device_placing", ['before', 'after'])
+@pytest.mark.parametrize("inference_type", ['cpu', 'single_GPU', 'DP', 'DDP'])
+def test_model_device_before_create_compressed_model(device_placing, inference_type):
+    if not torch.cuda.is_available() and not inference_type == 'cpu':
+        return
```

Comment:
Use `pytest.skip` instead and give a reason why the test is skipped, like it is being done in other tests.